src/Hangfire.Tags.PostgreSql/Friends.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
Isn't this file normally called `AssemblyInfo.cs`? At least, that's what I remember as the standard from .NET Framework days.

Reply:
it's a new way doing it for .netcore